NN.py

```diff
@@ -35,14 +35,47 @@ def __str__(self):
         return "Layer :\n  > nb_nodes = " + str(self.nb_nodes)
 
 
+class FeedForwardNeuralNetwork:
+    nb_layers = 0
 
-def gradient_descent(inputs, batch_size = 10, hidden_layer = 1, nb_nodes = 2, learning_rate= 0.01, momentum = 0.01, epoch = 10):
+    def __init__(self, nb_layers):
+        self.nb_layers = nb_layers
 
-    """
-    Returns sigmoid value of x
-    """
-    def sigmoid(x):
-        return 1 / (1 + math.exp(-x))
+    def predict(self, features, weights):
```

Comment:
this function is not needed, redundant

Reply:
> this function is not needed, redundant

Got it . Will be removed in the next commit.